exercises/G-numbers/exercise.js

```diff
@@ -1 +1,10 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numStu = "Number of Students: ";
+let numMen = "Number of Mentors: ";
+let numStuMen = "Total Number of Students and Mentors: ";
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+
+console.log(numStu + numberOfStudents);
```

Comment:
What you did is just fine, but also you don't have to declare parts of the string as a variable, especially if you're only using it once. So here you could simply do:
```javascript
console.log("Number of students: " + numberOfStudents);
```